app/setup/styleguide.styles.js

```diff
@@ -32,6 +34,28 @@ module.exports = {
         fontSize: 20
       }
     },
+    Heading: {
+      heading1: {
+        color: '#2c68a0',
+        width: '100%',
+        textAlign: 'center'
+      },
+      heading2: {
+        'backgroundColor': '#2c68a0',
+        'color': '#fff',
+        'width': 'calc(100% + 32px)',
+        'padding': '0px 15px',
+        'marginLeft': '-33px',
+        'zIndex': '1',
+        '& a': {
+          '&:hover': {
+            textDecoration: 'none',
+            color: '#fff !important',
+            cursor: 'pointer !important'
+          }
+        }
+      }
+    },
     ComponentsList: {
```

Comment:
- [ ] Can you increase the `margin-bottom` for the sections (the one with blue h2)?